src/_locales/ja/messages.json

```diff
@@ -0,0 +1,723 @@
+        "description": "Button in the placeholder shown in place of certain third-party (video, audio, commenting) widgets. Clicking this button will always allow the widget to load on this particular site."
+    },
+    "sync_intro": {
+        "message": "同期機能:<ul><li>Firefox/Chrome Syncが必要です。</li><li>アップロードを行うと、クラウドにある既存のPrivacy Badgerのデータを上書きします。</li><li>ダウンロードを行うと、Badgerが無効になっているサイトのリストを結合します。</li></ul>",
+        "description": "A brief explanation of how syncing works. Shown above the upload/download cloud data buttons under the Manage Data options page tab."
+    },
+    "upload": {
+        "message": "アップロード",
+        "description": "Button label for uploading data to the cloud on Manage Data options page tab."
+    },
+    "upload_cloud": {
+        "message": "保護機能を無効化したサイトをクラウドにエクスポート",
+        "description": "Button title for uploading data to the cloud on Manage Data options page tab."
+    },
+    "download_cloud": {
+        "message": "保護機能を無効化したサイトをクラウドからインポート",
+        "description": "Button title for downloading data from the cloud on Manage Data options page tab."
+    },
+    "download_cloud_success": {
+        "message": "クラウドデータのインポートに成功しました。",
+        "description": "Message displayed to the user after successfully merging cloud data."
+    },
+    "download_cloud_failure": {
+        "message": "クラウドデータをダウンロードできませんでした。",
+        "description": "Message displayed to the user after failing to download cloud data."
+    },
+    "upload_cloud_success": {
+        "message": "クラウドへのデータアップロードに成功しました。",
+        "description": "Message displayed to the user after successfully uploading local data."
+    },
+    "upload_cloud_failure": {
+        "message": "クラウドへのデータアップロードに失敗しました。",
+        "description": "Message displayed to the user after failing to upload local data."
+    },
+    "download_cloud_no_data": {
+        "message": "クラウドにダウンロード可能なデータがありません。",
+        "description": "'Error' message when there is on cloud data to download."
+    },
+    "share_title": {
+        "message": "友達に紹介する",
+        "description": "Title of the share overlay."
+    },
+    "popup_share_button": {
+        "message": "共有",
+        "description": "Tooltip that comes up when you hover over the share button in the upper right corner of the popup."
+    },
+    "share_tracker_header": {
+        "message": "Privacy Badgerは$COUNT$個の潜在的なトラッカーを$DOMAIN$でブロックしました:",
+        "description": "Header above the list of tracking domains in the share message.",
+        "placeholders": {
+            "count": {
+                "content": "$1",
+                "example": "15"
+            },
+            "domain": {
+                "content": "$2",
+                "example": "example.com"
+            }
+        }
+    },
+    "share_base_message": {
+        "message": "Privacy Badger (privacybadger.org) は、目に見えないトラッカーを自動的に学習してブロックするブラウザ拡張機能です。Privacy Badgerは、オンラインであなたの権利を守るために戦う非営利団体、Electronic Frontier Foundationによって作られています。",
```

Comment:
"Electronic Frontier Foundation" is translated in two other messages, but not in this one.